### ISSUE TITLE
Suggested DataCamp chapters vs. entire courses

### ISSUE THREAD

**Issue by andrewheiss** [moderndive/moderndive_book#71]
I've been assigning my students the suggested DataCamp chapters at the beginning of each ModernDive chapter to accompany MD. The experience went well with MD chapter 3, which suggested chapters 2 and 4 of David Robinson's "Introduction to the Tidyverse" DC course. Even though the suggestions had students skip chapters 1 and 3, most were fine and could understand what was going on without those chapters. Plus, I had them finish those DC chapters when they got to MD chapter 5 (since they're suggested there).

When students got to MD chapter 4 on tidy data, though, most panicked and struggled a ton (and many just gave up). The suggested DC resource is chapter 3 of @apreshill's "Working with data in the Tidyverse" course. Unlike David Robinson's course, which students were somehow able to skip around in with no problem, Alison's DC course is pedagogically designed to build on previous chapters, so students needed to have completed chapters 1 and 2 to work on 3 (at least that's what the few that went back and did 1–2 told me—I'm not 100% certain of that because I haven't actually taken her DC course 😬).

It might be helpful to *not* suggest specific chapters from DC courses, but whole courses instead. MD does this later on—chapter 10 suggests the whole courses on "Inference for Numerical Data" and "Inference for Categorical Data", for instance, instead of chapters.

**Comment by rudeboybert** [comment]
Hi @andrewheiss, thanks for the note
- Much like in DC's "Intro to Tidyverse" course, we debated starting with Data Wrangling and then doing Data Viz. However given the importance of Data Viz, we opted to start with it. I've tried both assigning DC "Intro to Tidyverse" as a whole (and hence having students see things in a different order) AND assigning DC Ch 2+4 and then DC 1+4 (and hence having students see things in the same order). It sounds like either approach could work, but with still a slight preference for assigning the whole chapter for smoothness
- Thanks for the note on "Working with data in the Tidyverse". The reason we only chose DC Ch 3 was because the DC course goes in a lot more detail than ModernDive Ch4 does. But given how the DC Chapters build on each other, we should probably scrap this. We'll definitely change that to read "Do the whole DC course, but note that it goes into a lot more detail than is necessary for ModernDive"
